Untitled.ipynb

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
Is this notebook supposed to be added? If so, change the name of the notebook to something more recognizable.

Reply:
No it's not :p I'll remove it